packages/ember-simple-auth-cookie-store/lib/simple-auth-cookie-store/stores/cookie.js

```diff
@@ -181,5 +208,52 @@ export default Base.extend({
       Ember.run.cancel(this._syncDataTimeout);
       this._syncDataTimeout = Ember.run.later(this, this.syncData, 500);
     }
+  },
+
+  /**
+    @method pageVisibilityInit
+    @private
+  */
+  pageVisibilityInit: function(){
+    var stateKey, eventKey, keys = {
+      hidden: "visibilitychange",
+      webkitHidden: "webkitvisibilitychange",
+      mozHidden: "mozvisibilitychange",
+      msHidden: "msvisibilitychange"
+    };
+    for (stateKey in keys) {
+      if (stateKey in document) {
+        eventKey = keys[stateKey];
+        break;
+      }
+    }
+    return function(c) {
+      if (c) {document.addEventListener(eventKey, c);}
+      return !document[stateKey];
+    };
+  },
+
+  /**
+    @method renew
+    @private
+  */
+  renew: function(data) {
+    data           = Ember.typeOf(data) === 'string' ? data : JSON.stringify(data || {});
+    var cachedExpirationTime = !!this.readExpirationTime() ? new Date().getTime() + this.readExpirationTime() * 1000 : null;
+    var expiration = !!this.cookieExpirationTime ? new Date().getTime() + this.cookieExpirationTime * 1000 : cachedExpirationTime;
+    this.write(data, expiration);
+  },
+
+  /**
+    @method renewExpiration
+    @private
+  */
+  renewExpiration: function() {
+    var data = this.restore();
+    if (!Ember.isEmpty(data) && data !== {} && this.pageVisibility()) {
+      this.renew(data);
+    }
+    Ember.run.cancel(this._renewExpirationTimeout);
+    this._renewExpirationTimeout = Ember.run.later(this, this.renewExpiration, 60000);
```

Comment:
This might cause problems when testing - better wrap that with `if (!Ember.testing) {…`